mage/query-modules/cpp/path.md

```diff
@@ -0,0 +1,811 @@
+---
+id: path
+title: path
+sidebar_label: path
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+  backgroundColor: color,
+  borderRadius: '2px',
+  color: '#fff',
+  padding: '0.2rem',
+}}>
+{children}
+</span>
+);
+
+The `path` module is a module for working with intricate connections in graph-like structures. It provides functions and procedures to navigate and analyze paths, helping users uncover meaningful insights from complex data relationships.
```

Comment:
```suggestion
The `path` module is a module for working with intricate connections in graph-like structures. It provides functions and procedures to navigate and analyze paths, helping users uncover meaningful insights from complex data relationships.
```

FYI this sounds like a politician wrote it :D You told me nothing :D At this point I have no idea what I will be able to do with this module or is it for me or my use case... I need to read on and check procedures and that's annoying :))